File: auditHub/src/main/java/com/thy/audithub/dto/ExportRequest.java
package com.thy.audithub.dto;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class ExportRequest {
    private List<IssueRowDto> rows = new ArrayList<>();
    private String mudurluk  = "";
    private String yearMonth = ""; // "2026-06"
    private boolean autoExportToDashboard = false;
}

File: Models/ExportRequest.cs
namespace AuditHub.Models;

public class ExportRequest
{
    public List<IssueRowDto> Rows { get; set; } = [];
    public string Mudurluk { get; set; } = "";
    public string YearMonth { get; set; } = "";
    public bool AutoExportToDashboard { get; set; }
}
